content/events/16-17/hashcode.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Hashcode
+description: Zeus organiseert een UGent Hashcode Hub
+created_at: 27-01-2017
+time: 23-02-2017 18:30
+location: A3, S9
+locationlink: Sterre,Ghent Belgium
+banner: https://hashcode.withgoogle.com/resources/images/hero_6.jpg
+image: https://zeus.ugent.be/zeuswpi/I9GAh7iy.png
+facebook: https://www.facebook.com/events/179465822503595/
+---
+
+# Google Hashcode
+Ook dit jaar organiseren we een hub voor de Google Hashcode. Dit is een jaarlijkse programmeerwedstrijd van Google waarin je in een team van 2 tot 4 personen een probleem voorgeschoteld krijgt die je mag oplossen in een programmeertaal naar keuze. De wedstrijd is gericht op studenten en professionals.
+
+De online kwalificatieronde vind plaats op 23 februari om 18:30. Je kan deelnemen met je team in onze hub _Ghent University_ in gebouw S9 op Campus Sterre, lokaal A3.
+
+De best scorende teams worden uitgenodigd bij Google Parijs op 1 april om mee te doen aan de finale ronde van de competitie.
+
+## Registratie
+Registreer je [hier](https://g.co/hashcode) om je in te schrijven. Daarna kun je ook onze hub selecteren en een team vormen. Vergeet ook niet om onze hub _Ghent University_ te selecteren. Hoe meer teams zich inschrijven voor 5 februari, hoe meer goodies we kunnen verdelen!
```

Comment:
Ik zou "Vergeet ook niet om onze hub _Ghent University_ te selecteren." in bold zetten, alsook meer nadruk leggen op het feit dat we basically al onze inschrijvingen de 5e februari willen. (ook al is dit niet realistisch)